bin/bcoin-cli

```diff
@@ -243,6 +260,7 @@ class CLI {
         this.log('  $ block [hash/height]: View block.');
         this.log('  $ header [hash/height]: View block header.');
         this.log('  $ filter [hash/height]: View filter');
+        this.log('  $ fee [target]: Estimate smart fee');
```

Comment:
nit: Also, this was missed in an earlier review for filter, however these all end with a period except these two.

Reply:
fixed.